sample-xml/app/src/test/java/co/nimblehq/sample/xml/ui/screens/second/SecondFragmentTest.kt

```diff
@@ -0,0 +1,65 @@
+package co.nimblehq.sample.xml.ui.screens.second
+
+import androidx.core.view.isVisible
+import co.nimblehq.sample.xml.R
+import co.nimblehq.sample.xml.databinding.FragmentSecondBinding
+import co.nimblehq.sample.xml.model.UiModel
+import co.nimblehq.sample.xml.test.getPrivateProperty
+import co.nimblehq.sample.xml.test.replace
+import co.nimblehq.sample.xml.ui.BaseFragmentTest
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.junit.*
+import java.util.*
+
+@HiltAndroidTest
+class SecondFragmentTest : BaseFragmentTest<SecondFragment, FragmentSecondBinding>() {
+
+    private val mockViewModel = mockk<SecondViewModel>(relaxed = true)
+    private val mockArgs = mockk<SecondFragmentArgs>(relaxed = true)
+
+    private val uiModel = UiModel(UUID.randomUUID().toString())
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
+
+        every { mockArgs.uiModel } returns uiModel
+    }
+
+    @Test
+    fun `When initializing fragment, it displays the text view`() {
```

Comment:
How about `launching` instead of `initializing`, since the latter could mean assigning value to a property?
```suggestion
    fun `When launching fragment, it displays the text view`() {
```